crates/subspace-node/src/cli.rs

```diff
@@ -14,22 +14,16 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-use sc_cli::RunCmd;
-use structopt::StructOpt;
-
-#[derive(Debug, StructOpt)]
-pub struct Cli {
-    #[structopt(subcommand)]
-    pub subcommand: Option<Subcommand>,
+//! Subspace CLI library.
 
-    #[structopt(flatten)]
-    pub run: RunCmd,
-}
+use structopt::StructOpt;
 
+#[allow(missing_docs)]
```

Comment:
Why not adding a comment instead of suppressing it for every struct?

Reply:
Was too lazy :P